src/components/Editor.vue

```diff
@@ -371,8 +371,8 @@ export default {
 				filePath: this.relativePath,
 				forceRecreate: this.forceRecreate,
 				serialize: this.isRichEditor
-					? () => createMarkdownSerializer(this.$editor.schema).serialize(this.$editor.state.doc)
-					: () => serializePlainText(this.$editor),
+					? (content) => createMarkdownSerializer(this.$editor.schema).serialize(content ?? this.$editor.state.doc)
+					: (content) => serializePlainText(content ?? this.$editor.state.doc),
```

Comment:
I think you are not making use of this (*anymore?*). Might be enough to keep the old less complex version - or use this in the `clipboardTextSerializer` below.

Reply:
We don't need to use it in the clipboardTextSerializer as we only overwrite it within the markdown extension. Otherwise tiptap will make use of this and i checked that both partial and full selection work as expected with plain text files.

Now we could just skip the parameter for the plain text serialize but I'd rather like to have consistent callback signatures.